src/main/java/graphql/execution/FieldCollectorParameters.java

```diff
@@ -26,15 +26,15 @@ public GraphQLSchema getGraphQLSchema() {
         return variables;
     }
 
-    public GraphQLObjectType getObjectType() {
-        return objectType;
+    public GraphQLFieldsContainer getFieldsContainer() {
+        return fieldsContainer;
     }
```

Comment:
Not true.  While the internal current code only checks objects, if you want to write "field selection capture code" then you need to be able to capture Interface objects.

Without this we cant capture all the fields in a selection as show by the tests